libafl/Cargo.toml

```diff
@@ -115,6 +115,8 @@ intel_pt = [
   "dep:nix",
   "dep:num_enum",
 ]
+# Save all the Intel PT raw traces to files, use only for debug
```

Comment:
This needs two `##` to show up in the docs

Reply:
Fixed, thanks!